vidify/player/web.py

```diff
@@ -0,0 +1,83 @@
+"""
+"""
+
+import logging
+from typing import Optional
+from threading import Thread
+from webbrowser import open
+from threading import Timer, Thread
+
+try:
+    import flask
+except OSError:
+    raise ImportError("Flask is not installed")
+
+from vidify import Platform, CURRENT_PLATFORM
+from vidify.player.generic import PlayerBase
+from PyQt5.QtCore import QObject
+import sys
+import os
+
+PORT = 9999
+ROOT_URL = 'http://localhost:{}/video'.format(PORT)
+DEFAULT_VIDEO_ID = "GfKs8oNP9m8"
+class WebPlayer(PlayerBase, QObject):
+    DIRECT_URL = False
+    app = None
+    currentMedia = ""
+    isPlaying = True
+    def __init__(self, vlc_args: Optional[str] = None) -> None:
+        super().__init__()
+        print(__name__)
+        self.app = flask.Flask("flask_web_server", template_folder="vidify/player/templates", static_folder="vidify/player/static")
```

Comment:
For this you can make a `vidify.player.web` module and put `templates`, `static` and this `web.py` in there.